src/widgets/about-course/ui/about-course-grid/about-course-grid.module.scss

```diff
@@ -1,68 +1,54 @@
-.about-grid {
+.about-course-grid {
   display: grid;
   grid-template-areas:
     'a a b'
     'c d e';
   grid-template-columns: 1fr 1fr 1fr;
-  gap: 30px;
+  gap: 32px 30px;
 
-  margin-top: 24px;
+  margin-bottom: 16px;
+
+  .grid-item {
+    display: flex;
+    flex-direction: column;
+    gap: 18px;
 
-  .item {
     width: 100%;
     padding: 24px;
 
-    font-size: 18px;
-    line-height: 28px;
-    color: $color-gray-600;
-    letter-spacing: 0;
-
     background-color: $color-yellow-50;
-    border: 1px solid rgb(255 219 32 / 7.8%);
+    border: 1px solid hsla(from $color-yellow h s l/ $opacity-8);
     border-radius: 12px;
 
-    & > .item-title {
+    .item-title {
       display: flex;
       gap: 16px;
       align-items: center;
-      justify-content: flex-start;
-
-      margin-bottom: 10px;
 
-      img {
+      .grid-icon {
         max-width: 50px;
         max-height: 40px;
       }
-
-      > h2 {
-        margin: 0;
-        font-size: 20px;
-        font-weight: $font-weight-medium;
-      }
-    }
-
-    @include media-laptop {
-      font-size: 16px;
     }
   }
 
-  .item:nth-child(1) {
+  .grid-item:nth-child(1) {
     grid-area: a;
   }
 
-  .item:nth-child(2) {
+  .grid-item:nth-child(2) {
     grid-area: b;
   }
 
-  .item:nth-child(3) {
+  .grid-item:nth-child(3) {
     grid-area: c;
   }
 
-  .item:nth-child(4) {
+  .grid-item:nth-child(4) {
     grid-area: d;
   }
 
-  .item:nth-child(5) {
+  .grid-item:nth-child(5) {
     grid-area: e;
   }
 
```

Comment:
would be better to use this way

<img width="258" alt="Screenshot 2024-09-30 at 18 28 04" src="https://github.com/user-attachments/assets/a93509d2-62c6-4b82-b984-3f139314ef6a">